semperfiwebdesign/all-in-one-seo-pack
# Restrict .ui-sortable-handle span {} to aioseop options

Issue #2569.

## Proposed changes

Restricts the .ui-sortable-handle span {} class to our own options as this could affect other plugin's meta boxes in wp_admin.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Check all menus and make sure that the header of each section still has the same style that was introduced in AIOSEOP v3.0. Also, have a look at the code so you understand the change that was made.
